platform-sdk/platform-apps/tests/PlatformTestingTool/src/main/java/com/swirlds/demo/platform/PlatformTestingToolState.java

```diff
@@ -1608,4 +1575,9 @@ private static class ChildIndices {
 
         public static final int CHILD_COUNT = 11;
     }
+
+    @Override
+    public void preHandle(final Event event) {
+        event.forEachTransaction(this::preHandleTransaction);
```

Comment:
Should the value of `initialized` be checked here, like in `handleConsensusRound`?

Reply:
It was not previously being checked. This implementation is a direct copy-paste from `PlatformTestingState2.java`, which was collapsed into this file.

My general approach for PTT is to avoid changing business logic at all costs, since if I break something I have to spend time debugging it. This is especially true when the deletion of this testing app is on the horizon.

Are you ok leaving this the way it is for now?

Reply:
Sounds good to me